.babelrc

```diff
@@ -1,13 +1,73 @@
 {
-  "presets": ["react", "env", "stage-0"],
+  "presets": [
+    "@babel/preset-react",
+    "@babel/preset-env"
+  ],
   "env": {
     "production": {
       "plugins": [
         "transform-react-remove-prop-types",
-        "transform-react-constant-elements",
-        "transform-react-inline-elements"
+        "@babel/plugin-transform-react-constant-elements",
+        "@babel/plugin-transform-react-inline-elements",
+        "@babel/plugin-syntax-dynamic-import",
```

Comment:
The [`@babel/plugin-syntax-dynamic-import` documentation](https://babeljs.io/docs/en/babel-plugin-syntax-dynamic-import#working-with-webpack-and-babel-preset-env) says:

> Currently, @babel/preset-env is unaware that using import() with Webpack relies on Promise internally. Environments which do not have builtin support for Promise, like Internet Explorer, will require both the promise and iterator polyfills be added manually.

Reply:
i think this mostly affects IE support. to get this to work, does `core-js` need to be added as a dependency? i fell into a hole reading about `@babel/preset-env` with [`useBuiltIns: "usage"`](https://babeljs.io/docs/en/babel-preset-env#usebuiltins-usage-experimental), but can't figure out whether using this feature affects this plugin.

Reply:
Yes, I think setting `useBuiltIns` to anything other than `false` will insert `core-js` imports for the required polyfills.

`core-js` seems to already be included in `node_modules` because of the following dependencies:

  `@babel/preset-env` ▶️  `core-js-compat` ▶️ `core-js`

Reply:
okay, i just pushed a change to the webpack configs to support this.